official/transformer/v2/transformer_benchmark.py

```diff
@@ -150,6 +151,54 @@ def benchmark_1_gpu(self):
                                    bleu_min=25.3,
                                    bleu_max=26)
 
+  def benchmark_8_gpu(self):
+    """Benchmark 8 gpu.
+
+      The paper uses 8 GPUs and a much larger effective batch size, this is will
+      not converge to the 27.3 BLEU (uncased) SOTA.
+    """
+    self._setup()
+    FLAGS.num_gpus = 8
+    FLAGS.data_dir = self.train_data_dir
+    FLAGS.vocab_file = self.vocab_file
+    # Sets values directly to avoid validation check.
+    FLAGS['bleu_source'].value = self.bleu_source
+    FLAGS['bleu_ref'].value = self.bleu_ref
+    FLAGS.param_set = 'base'
+    FLAGS.batch_size = 32768
+    FLAGS.train_steps = 200000
+    FLAGS.static_batch = True
+    FLAGS.steps_between_evals = 5000
+    FLAGS.model_dir = self._get_model_dir('benchmark_8_gpu')
+    self._run_and_report_benchmark(total_batch_size=FLAGS.batch_size,
+                                   log_steps=FLAGS.log_steps,
+                                   bleu_min=25.3,
+                                   bleu_max=26)
+
+  def benchmark_big_8_gpu(self):
```

Comment:
Make a new class: TransformerBigKerasAccuracy and then just use `benchmark_8_gpu`

I look for the word  `TransformerBig` as a way to know the model when I do reporting.